src/lib/components/NostriChat.svelte

```diff
@@ -0,0 +1,33 @@
+<script>
+    const groupID = "<GROUP_ID_IN_HEX_FORMAT>";
```

Comment:
Yup – in this case you need to replace the group id in hex with the hex of the actually NIP-28 chat room. You can create those in this app: https://www.nostrchat.io/

Reply:
Very cool! Thank you for sharing how that is done.